src/middlewared/middlewared/plugins/network.py

```diff
@@ -465,16 +465,40 @@ def ipv4_in_use(self):
         Returns:
             list: A list with all IPv4 in use, or an empty list.
         """
-        list_of_ipv4 = []
+        list_of_ipv4 = self._ip_in_use()
+
+        return list_of_ipv4
+
+    def _ip_in_use(self, ip4=True):
```

Comment:
I was thinking this could be a public method making it return ipv4 _and_ ipv6.

I can imagine scenarios we would want both and making two different calls is not ideal.